server/controllers/finance/reports/collection_capacity/report.handlebars

```diff
@@ -0,0 +1,54 @@
+{{> head title="REPORT.COLLECTION_CAPACITY.TITLE"}}
+
+<body>
+
+<div class="container">
+  {{> header}}
+
+  <!-- body  -->
+  <div class="row">
+    <div class="col-xs-12">
+
+      <!-- page title  -->
+      <h3 class="text-center text-uppercase">
+        <strong>{{translate 'REPORT.COLLECTION_CAPACITY.TITLE'}}</strong> <br>
+        <small>{{account.number}} - {{account.label}}</small><br>
```

Comment:
Let's remove the references to accounts - we want to see the balance in all cashboxes that day.